lib/src/base_validator.dart

```diff
@@ -0,0 +1,47 @@
+/// Base class for all validators.
+abstract class BaseValidator<T> {
+  /// Creates a new instance of the validator.
+  const BaseValidator({String? errorText, this.checkNullOrEmpty = true})
+      : _errorText = errorText;
+
+  /// Backing field for [errorText].
+  final String? _errorText;
+
+  /// {@template base_validator_error_text}
+  /// The error message returned if the value is invalid.
+  /// {@endtemplate}
+  String get errorText => _errorText ?? translatedErrorText;
+
+  /// The translated error message returned if the value is invalid.
+  String get translatedErrorText;
```

Comment:
I don't sure about this property. I think that is better that no add translatedErrorText and only use errorText.
With these approach, can pass translatedErrorText on `form_builder_validators.dart` and this class (and their implementations) will be pure Dart.

Reply:
I've experimented with a couple of ways but i'm quite sure this is the best approach. This way we make sure the built-in translation is always available on the correct validator. It streamlines code styling and makes new validators follow the standards.